src/jit/regset.cpp

```diff
@@ -1604,7 +1604,7 @@ void RegSet::rsSpillTree(regNumber reg, GenTreePtr tree, unsigned regIdx /* =0 *
 #endif // _TARGET_ARM_
     else
     {
-        assert(!varTypeIsMultiReg(tree));
+        assert(!varTypeIsMultiReg(tree) || (m_rsCompiler->opts.compUseSoftFP && treeType == TYP_LONG));
```

Comment:
I don't think this is correct. How does it know which have of the double/long is being spilled? I think it needs to be handled similarly to the other multiReg ops.